library/src/main/java/com/okta/appauth/android/OktaAppAuth.java

```diff
@@ -741,7 +744,7 @@ private void recreateAuthorizationService(Context context) {
      */
     AuthorizationService createAuthorizationServiceIfNeeded() {
         if (mAuthService.get() == null) {
-            recreateAuthorizationService(mContext);
+            recreateAuthorizationService(mContext.get());
```

Comment:
does `recreateAuthorizationService` work if the context is null here?

Reply:
It won't work if context is null. This only happens when the Activity is closed. This should never happen but I'll add a null check and throw a exception to init OktaAppAuth again.